### PR TITLE
Repair implicit flow sample

### PR DESCRIPTION
**Pull request by shaunluttin**
https://github.com/openiddict/openiddict-samples/issues/18

### CODE REVIEW

File: samples/ImplicitFlow/AureliaApp/package.json [hunk 1]
```diff
@@ -1,70 +1,51 @@
 {
-  "name": "aurelia-openiddict",
-  "description": "A sample that shows aurelia-openid-client working with OpenIddict.",
+  "name": "demo",
```
Comment (PinpointTownes) [inline samples/ImplicitFlow/AureliaApp/package.json]: Please fix that.

File: samples/ImplicitFlow/AureliaApp/src/home.ts [hunk 2]
```diff
@@ -66,13 +51,22 @@ export class Login {
...
-
-        let leftPart = serverNum === 1
-            ? "http://localhost:5001"
-            : "http://localhost:5002";
-
-        let path = isPrivate ? "private" : "public";
+        let leftPart: string;
+        let path: string;
+
+        if (window.location.hostname.startsWith("localhost")) {
```
Comment (PinpointTownes) [inline samples/ImplicitFlow/AureliaApp/src/home.ts]: Please revert that too.

File: samples/ImplicitFlow/AureliaApp/src/open-id-connect-configuration.ts [hunk 3]
```diff
@@ -1,18 +1,29 @@
-import { OpenIdConnectConfiguration, UserManagerSettings } from "aurelia-open-id-connect";
+import { OpenIdConnectConfiguration, UserManagerSettings, WebStorageStateStore } from "aurelia-open-id-connect";
+
+let isDevelopment = window.location.host.startsWith("localhost");
+
+let authority: string = isDevelopment
```
Comment (PinpointTownes) [inline samples/ImplicitFlow/AureliaApp/src/open-id-connect-configuration.ts]: Revert.

File: samples/ImplicitFlow/AureliaApp/src/open-id-connect-configuration.ts [hunk 4]
```diff
@@ -1,18 +1,29 @@
...
+    ? "http://localhost:12345"
+    : "https://zamboni-auth.azurewebsites.net";
+
+let host: string = isDevelopment
```
Comment (PinpointTownes) [inline samples/ImplicitFlow/AureliaApp/src/open-id-connect-configuration.ts]: Revert.

File: samples/ImplicitFlow/AureliaApp/test/aurelia-karma.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
+(function(global) {
```
Comment (PinpointTownes) [inline samples/ImplicitFlow/AureliaApp/test/aurelia-karma.js]: There are no tests in this sample, right?

**Comment by shaunluttin** [comment]
@PinpointTownes I think you will like this much better now. Instead of 50 changed files it has only about four. That is much easier on the reviewer.

**Pull request closed without merging**